snippets/sdp/gpio/icbmsg/snippet.yml

```diff
@@ -7,11 +7,15 @@
 name: sdp-gpio-icbmsg
 
 boards:
-  nrf54l15dk/nrf54l15/cpuapp:
+  /.*/cpuapp/:
     append:
-      EXTRA_DTC_OVERLAY_FILE: boards/nrf54l15dk_nrf54l15_cpuapp.overlay
       EXTRA_CONF_FILE: app.conf
-  nrf54l15dk/nrf54l15/cpuflpr:
+  /.*/cpuflpr/:
     append:
-      EXTRA_DTC_OVERLAY_FILE: boards/nrf54l15dk_nrf54l15_cpuflpr.overlay
       EXTRA_CONF_FILE: flpr.conf
```

Comment:
should be moved into the parts below otherwise someone can use this snippet with an nrf5340 and it will include app.conf